lib/vSphere/action/get_ssh_info.rb

```diff
@@ -13,21 +13,27 @@ def initialize(app, _env)
 
         def call(env)
           env[:machine_ssh_info] = get_ssh_info(env[:vSphere_connection], env[:machine])
-
           @app.call env
         end
 
         private
 
+        def filter_guest_nic(vm, machine)
+          return vm.guest.ipAddress unless machine.provider_config.real_nic_ip
+          adapters = vm.guest.net.select { |g| g.deviceConfigId > 0 }.map { |g| g.ipAddress[0] }
+          fail Errors::VSphereError.new, 'real_nic_ip filtering set with multiple valid VM interfaces available' if adapters.size > 1
```

Comment:
This error message string should be defined and pulled from [`locales/en.yml`](https://github.com/nsidc/vagrant-vsphere/blob/master/locales/en.yml)